CustomeDB/CustomeDB.cpp

```diff
@@ -4,19 +4,28 @@
 #include <iostream>
 using namespace std;
 
+class Item {
+public:
+    string name;
+    int price;
+    int id;
+};
+
 class Node {
 public:
-    int data;
+    Item data;
     Node* next;
 };
 
 class LinkedList {
     Node* head = NULL;
 public:
-    void addAtEnd(int val) {
+    void addAtEnd(int id, string name, int price) {
 
         Node* new_node = new Node();
-        new_node->data = val;
```

Comment:
the implementation of linked list should'nt change just because ur using object,
the data copy and deletion should be handled by class for applying OOP concepts.

Reply:
fixed.